dgl_ptm/dgl_ptm/config.yaml

```diff
@@ -12,9 +12,38 @@ steering_parameters:
   - all
   npath: ./agent_data.zarr
   ratio: 0.1
-  wealth_method: weighted_transfer
-  homophily_parameter: 0.69
-  characteristic_distance: 35
+  wealth_method: singular_transfer
+  income_method: income_generation
+  consume_method: fitted_consumption
+  adapt_m:  
+    - 0.
+    - 0.5
+    - 0.9
+  adapt_cost:
+    - 0.
+    - 0.25
+    - 0.45
+  depreciation: 0.6
+  discount: 0.95
+  m_theta_dist': 
+    type: multinomial
+    parameters:
+    - [0.02 ,0.03, 0.05, 0.9]
+    - [0.7, 0.8, 0.9, 1]
+    round:  False
+    decimals: None
+  weight_a: 0.69
```

Comment:
nomenclature mismatcg weight_a/_b and homophily_parameter characteristic_distance